### ISSUE TITLE
Remove tachyons in favor of only normalize.css

### ISSUE THREAD

**Issue by AntonTrollback** [choojs/create-choo-app#56]
Hello!
I'm suggesting that we remove tachyons as a core dependency and stick to something less opinionated, only normalize.css. Or no css at all.

Not trying to start a discussion about how to structure css 😇

**Comment by yoshuawuyts** [comment]
> Not trying to start a discussion about how to structure css

Haha, isn't that exactly what this is? :P Don't mind it tho, it's good to talk about things like these.

My reasoning for including Tachyons is that it allows you to get started straight away. The first thing people see is a guide on how to get started, and some base files. Tachyons allows this to happen - so regardless we need to include something.

**Comment by chrisdwheatley** [comment]
I'm kind of biased as I use Tachyons a lot and love it!

However, for someone who has their own preferred CSS solution I can understand why prescribing Tachyons might not be such a good fit.

Looking at similar projects such as `create-react-app` and `preact-cli` they offer some CSS tooling setup but not much else.

I guess what it comes down to is what the goal of `create-choo-app` is and does prescribing Tachyons help towards that goal?

**Comment by ghost** [comment]
I believe the reasonable way to compromise with this issue is to add a guide on how to replace tachyons with something else if desired? Keep in mind that "including tachyons" means just having _one dependency_ and _one line of code_, so it should be fairly easy to replace.

Although I can also see how tachyons is maybe not the most approachable solution for newcomers to choo and modular architecture in general, as it's not exactly the most approachable framework in the world, when compared to something like Bootstrap.

**Comment by tnguyen14** [comment]
I agree with @oe. I just started out using choo today, and having references to how to do things in different ways would be the most helpful way to learn what choo is, what it does and doesn't. Having conventions is helpful, but it's equally helpful to know how to change that convention as necessary.

And other than `sheetify`, is there another way to include CSS, perhaps as an external file? That is probably the most common way of using CSS still.

**Comment by AntonTrollback** [comment]
Yep. We should handle how newcomers (me included) will think "Where am I supposed to write my CSS?".

I propose that we add an explicit example, a file, for where CSS can be written. That file can then contain the few lines of CSS for laying out the main view, matching the choo design identity.

🤔 Perhaps it could be in the form of a component that exports only the CSS for that view, in a general way such as `base` or `example`. That component could then import normalize.css if we'd like.

🤔 Or call it something like `introduction` or `welcome` and have it contain both the CSS and the content for the main view.

Going further, the example button could be abstracted to a `button` component that includes the markup and the CSS for it, working as an example of components as well as how one can style them. Relies on https://github.com/choojs/choo/pull/639.

> The first thing people see is a guide on how to get started, and some base files. Tachyons allows this to happen - so regardless we need to include something.

Before getting started, as a front-end dev, I need to know how to create html, sprinkle interaction on top of it, and how to apply my styles, CSS that follows design resources and principles we share in the workplace.

**Comment by jondashkyle** [comment]
Yeah I agree @AntonTrollback. Whenever I use `create-choo-app` I go and remove tachyons and setup my own CSS file. Including a little reset (`nanoreset`?) to show an example of requiring an npm module plus a little `index.css` could be great.

**Comment by chrisdwheatley** [comment]
There's now a brief note about removing Tachyons in the [readme](https://github.com/choojs/create-choo-app#removing-dependencies).

If anyone wants to expand upon that feel free to.

**Comment by joseph-long** [comment]
I, for one, would be interested in an example of how to include CSS in a more modular / component-y way than just putting my custom CSS in `assets/`.
